scan_to_paperless/process_schema.json

```diff
@@ -506,6 +506,37 @@
               "default": 6
             }
           }
+        },
+        "rest_upload": {
+          "type": "object",
+          "title": "REST upload",
+          "description": "Upload the final pdf via Paperless REST API",
+          "properties": {
+            "enable": {
+              "type": "boolean",
+              "default": false,
+              "description": "Enable the upload of the PDF via REST API",
+              "title": "REST upload Enable"
+            },
+            "api_url": {
+              "type": "string",
+              "default": "PLEASE CONFIGURE api_url",
```

Comment:
I added this default, and the token one, so that if you don't configure one of the two, the error message will contain it  and it's evident what's missing.
It is a somewhat lazy way of avoiding to write proper checks and error messages :-)